### ISSUE TITLE
NVMe-TCP target:  cannot open NVMe partition in filesystem autotest

### ISSUE THREAD

**Issue by shuhei-matsumoto** [spdk/spdk#1166]
Please use the issue tracker only for reporting suspected issues.

See [The SPDK Community Page](http://www.spdk.io/community/) for other SPDK communications channels.

The following has been observed intermittently.

```
************************************
START TEST filesystem_xfs
************************************
 03:25:13	# xtrace_enable
 03:25:13	# nvmf_filesystem_create xfs nvme0n1
 03:25:13	# fstype=xfs
 03:25:13	# nvme_name=nvme0n1
 03:25:13	# '[' xfs = ext4 ']'
 03:25:13	# force=-f
 03:25:13	# mkfs.xfs -f /dev/nvme0n1p1
mkfs.xfs: cannot open /dev/nvme0n1p1: Device or resource busy
  03:25:13	# trap - ERR
  03:25:13	# print_backtrace
  03:25:13	# [[ ehxBE =~ e ]]
  03:25:13	# xtrace_disable
  03:25:13	# '[' no '!=' yes ']'
  03:25:13	# PREV_BASH_OPTS=ehxBE
  03:25:13	# [[ ehxBE == *\x* ]]
  03:25:13	# XTRACE_DISABLED=yes
  03:25:13	# set +x
========== Backtrace start: ==========
```

## Expected Behavior
No error

## Current Behavior
The above

## Possible Solution
Ben already provided a fix.
https://review.gerrithub.io/c/spdk/spdk/+/482926

## Steps to Reproduce
Run filesysttem test with TCP transport.

## Context (Environment including OS version, SPDK version, etc.)
Latest SPDK master

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @shuhei-matsumoto. log: https://dqtibwqq6s6ux.cloudfront.net/public_build/autotest-per-patch_58360.html

**Comment by peluse** [comment]
@shuhei-matsumoto can you confirm this is fixed and close it if so?

**Comment by benlwalker** [comment]
This is fixed by commit c85a3d105
